app-misc/diff-so-fancy/diff_so_fancy-1.4.3.recipe

```diff
@@ -0,0 +1,32 @@
+SUMMARY="Pretty diff generator"
+DESCRIPTION="diff-so-fancy strives to make your diffs human readable instead of machine readable. This helps improve code quality and helps you spot defects faster.
+
+Configure git to use diff-so-fancy for all diff output:
+
+git config --global core.pager \"diff-so-fancy | less --tabs=4 -RFX\"
+git config --global interactive.diffFilter \"diff-so-fancy --patch\""
+HOMEPAGE="https://github.com/so-fancy/diff-so-fancy"
+COPYRIGHT="2016 So Fancy team"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/so-fancy/diff-so-fancy/releases/download/v$portVersion/diff-so-fancy#noarchive"
+CHECKSUM_SHA256="de90659705436cf17d9a8aaad7012c9ed82d7f16eadf1ecee3d6badb0717b73f"
+
+DISABLE_SOURCE_PACKAGE="yes"
+ARCHITECTURES="all"
+
+PROVIDES="
+	diff_so_fancy
+	cmd:diff_so_fancy
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
```suggestion
	haiku
```
`$secondaryArchSuffix` can be dropped since this package can be built for all primary architectures